### ISSUE TITLE
Error: Cannot read property 'forEach' of undefined

### ISSUE THREAD

**Issue by lll000111** [feross/spoof#23]
Windows 10, node.js 10.9.0

```
$ spoof list
Error: Cannot read property 'forEach' of undefined
```

No stack trace, only that message.

```
$ spoof version
2.0.1
```

Also:

```
$ spoof reset wi-fi
Error: Could not find device for wi-fi

$ spoof randomize wi-fi
Error: Could not find device for wi-fi
```

(Same when using "Wi-Fi" as ID)

My network:

```
$ ipconfig

Windows IP Configuration


Wireless LAN adapter Local Area Connection* 15:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Wireless LAN adapter Local Area Connection* 16:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :

Wireless LAN adapter Wi-Fi:

   Connection-specific DNS Suffix  . : fritz.box
   IPv6 Address. . . . . . . . . . . : 2003:ea:93d0:f500:1850:c080:15ca:a08b
   Temporary IPv6 Address. . . . . . : 2003:ea:93d0:f500:f409:fa1f:aec5:5195
   Link-local IPv6 Address . . . . . : fe80::1850:c080:15ca:a08b%21
   IPv4 Address. . . . . . . . . . . : 192.168.178.28
   Subnet Mask . . . . . . . . . . . : 255.255.255.0
   Default Gateway . . . . . . . . . : fe80::3631:c4ff:fefb:6967%21
                                       192.168.178.1

Ethernet adapter Bluetooth Network Connection 6:

   Media State . . . . . . . . . . . : Media disconnected
   Connection-specific DNS Suffix  . :
```

**Comment by ShinduHub** [comment]
Same over here and also Windows 10 with  nodejs v11.11.0:
```
$ spoof list
Error: Cannot read property 'forEach' of undefined
```

**Comment by choleaoum** [comment]
Same issue, version v10.15.3, windows server 2012 r2. Thanks!

**Comment by feross** [comment]
Hi folks, sorry that you're experiencing an issue. Since I don't have a Windows machine, I recommend someone take a look at the code to find the issue. Once you find it, send a pull request and I'd be more than happy to review it and merge it. Cheers!

**Comment by 3m1n470r** [comment]
I'm having same problems. Di anybody find out how to fix this issue?

**Comment by feross** [comment]
Fixed by #32 and released as 2.0.3.
